.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@ on: [push, pull_request]
 name: Continuous integration
 
 env:
-  MSRV: 1.70.0
+  MSRV: 1.77.0
```

Comment:
Interesting that we need this because of clap. From the CI output:
```
error: package `clap v4.5.13` cannot be built because it requires rustc 1.74 or newer, while the currently active rustc version is 1.70.0
Either upgrade to rustc 1.74 or newer, or use
cargo update -p clap@4.5.13 --precise ver
where `ver` is the latest version of `clap` supporting rustc 1.70.0
Error: The process '/home/runner/.cargo/bin/cargo' failed with exit code 101
```
But it’s fine, we can update to rust 1.77.0, it went out almost 6 months ago at this point and if clap needs it, lots of our users will likely need it as well because of clap. So yeah, let’s bump the MSRV.

Reply:
In first time I bumped MSRV to 1.74.0 according to the CI output. But `rusqlite` in 0.32 uses a ructc-feature `c_literal` ([commit](https://github.com/rusqlite/rusqlite/commit/ed7434e95852f14e3a8683927a136e4daa8ffa9e)), which has been stable in 1.77.0. So then I bumped it to 1.77.0